feature_tests/java/somelib/src/main/java/dev/diplomattest/somelib/Opaque.java

```diff
@@ -0,0 +1,67 @@
+package dev.diplomattest.somelib;
+
+import dev.diplomattest.somelib.ntv.*;
+
+
+import java.lang.foreign.Arena;
+import java.lang.foreign.MemorySegment;
+import java.lang.ref.Cleaner;
+import java.nio.charset.StandardCharsets;
+
+
+public class Opaque {
```

Comment:
Although the tool doesn't actually generate code, the class body has now been been copied from an insta snapshot.

Reply:
☝️ no longer valid 😝